tests/acceptance/CustomUnsubscribePageCest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class CustomUnsubscribePageCest {
+  function createCustomUnsubscribePage(\AcceptanceTester $I) {
+    $I->wantTo('Create page with MP subscriber shortcode');
+    $pageTitle='SorryToSeeYouGo';
+    $pageContent='[mailpoet_manage\ text=\"Manage\ your\ subscription\"]';
```

Comment:
You don't have to add those backslashes here but you can use `escapeshellarg()` so that it could be written like that:
`$pageContent = escapeshellarg('[mailpoet_manage text="Manage your subscription"]');`
Then you may also want to put the page text into a variable so that it is not repeated on two places:
```php
    $pageText = 'Manage your subscription';
    $pageContent = escapeshellarg("[mailpoet_manage text=\"$pageText\"]");
    ...
    ...
    $I->waitForText($pageText);
```

Reply:
This is extremely helpful, thanks! The test is so much easier to read now.